bin/console

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env ruby
+
+require "bundler/setup"
+require "hanami/setup"
+require "pry"
+
+Hanami.init
+Pry.start
```

Comment:
I added this since it was helpful to me while playing around. Later we can convert this to the proper console that I've started establishing in the PR to hanami/hanami.

FWIW this also shows the use of `require "hanami/setup"` for automatically loading key Hanami config files (config/application, config/routes) without having to do any manual labor :) It also shows usage of `Hanami.init` partially boot the hanami application (i.e. just detecting the slices, not booting the app or finalising any containers).